src/native/managed/cdac/Microsoft.Diagnostics.DataContractReader.Contracts/Contracts/RuntimeTypeSystem_1.cs

```diff
@@ -128,6 +130,21 @@ private static uint ComputeToken(Target target, Data.MethodDesc desc, Data.Metho
             return EcmaMetadataUtils.CreateMethodDef(tokenRange | tokenRemainder);
         }
 
+        private static uint ComputeSize(Target target, Data.MethodDesc desc)
+        {
+            // Size of the MethodDesc is variable, read it from the targets lookup table
+            // See MethodDesc::SizeOf in method.cpp for details
+            // TODO(cdac): make sure this value is stored in minidumps
+            TargetPointer methodDescSizeTable = target.ReadGlobalPointer(Constants.Globals.MethodDescSizeTable);
+
+            ushort arrayOffset = (ushort)(desc.Flags & (ushort)(
+                MethodDescFlags_1.MethodDescFlags.ClassificationMask |
+                MethodDescFlags_1.MethodDescFlags.HasNonVtableSlot |
+                MethodDescFlags_1.MethodDescFlags.HasMethodImpl |
+                MethodDescFlags_1.MethodDescFlags.HasNativeCodeSlot));
```

Comment:
This is out of date. The merge of the async logic has added a new flag.

Reply:
In general, you need to review the updates done by async to see if there are any other changes we need, and if we need to bump a contract version.